scrcpy/src/main/java/io/github/miuzarte/scrcpyforandroid/widgets/DeviceWidgets.kt

```
@@ -0,0 +1,1194 @@
+
+private fun presetIndexFromInput(raw: String, presets: List<Int>): Int {
+    if (raw.isBlank()) return 0
+    val value = raw.toIntOrNull() ?: return 0
+    val exact = presets.indexOf(value)
+    if (exact >= 0) return exact
+    val nearest = presets.withIndex().minByOrNull { (_, preset) -> kotlin.math.abs(preset - value) }
+    return nearest?.index ?: 0
+}
+
+@SuppressLint("DefaultLocale")
+private fun formatBitRate(value: Float): String = String.format("%.1f", value)
```

detekt (1.23.8)

[warning] String.format("%.1f", value) uses implicitly default locale for string formatting.

(detekt.potential-bugs.ImplicitDefaultLocale)